### PR TITLE
Resolves #184: Update Readme and add install script for Linux

### LINKED ISSUE #184: Install instructions for linux/ubuntu
Hello,
i managed to make nibbler work, but it was troublesome. Maybe add instructions to the readme

make desktop shortcut:

```
cd ~/Downloads
wget -c https://github.com/rooklift/nibbler/releases/download/v2.2.1/nibbler-2.2.1-linux.zip
sudo unzip nibbler-2.2.1-linux.zip -d /opt/
sudo chmod +x /opt/nibbler-2.2.1-linux/nibbler
sudo mkdir -p /usr/local/share/applications

cat <<EOF | sudo tee /usr/local/share/applications/nibbler.desktop
[Desktop Entry]
Type=Application
Version=1.0
Name=Nibbler
Icon=/opt/nibbler-2.2.1-linux/resources/app/pieces/K.png
Exec=/opt/nibbler-2.2.1-linux/nibbler
Terminal=false
StartupNotify=false
Categories=Game;BoardGame;
EOF
```
Then copy nibbler.desktop from usr/local/share/applications to desktop -> right click -> allow start

Create a symlink

`sudo ln -s /opt/nibbler-2.2.1-linux/nibbler /usr/local/bin/nibbler`

then you can open it by typing "nibbler" into terminal.

### PR DESCRIPTION
**Pull request by cyrillkuettel**
This script completely automates the installation for Linux users. The latest version is detected from the [latest releases](https://github.com/rooklift/nibbler/releases/tag/v2.3.6) 

Furthermore, the script will ask if you want to create a Desktop shortcut. The created shortcut looks like this:

Screenshot from 2022-08-15 23-58-27

* Tested on Ubuntu 18.04 and 22.04

### CODE REVIEW

**Comment by rooklift** [comment]
Hmm. It's difficult for me to accept this, because I barely speak bash and/or linux commands and it's hard for me to know that everything about this is entirely safe.

**Comment by cyrillkuettel** [comment]
> Hmm. It's difficult for me to accept this, because I barely speak bash and/or linux commands and it's hard for me to know that everything about this is entirely safe.

Maybe you know someone who speaks bash / linux and they can verify it? Although it may seem verbose, it's mechanism is really quite simple. It just downloads the .zip and extracts it to the common location.

**Comment by cyrillkuettel** [comment]
Sure thing. I put in in the `src`. 

Thanks for this wonderful software!

**Comment by rooklift** [comment]
As you suggested, I had a Linux person look it over. They suggested that `~/Downloads` might not exist and `/tmp` might be better. Thoughts?

**Comment by teck45** [comment]
Looks well. Couple of ideas. You are using sudo couple of times to create folders inside applications folder. Good idea to add echo (print for people not familiar with bash) telling that sudo is needed to make folder XX. And echo of command, then command. Example oil your code: 
echo "Unzipping to $LOCATION"
sudo unzip -qq ${ZIP_NAME} -d /opt/

what can be done
echo "Unzipping to $LOCATION, sudo needed"
echo sudo unzip -qq ${ZIP_NAME} -d /opt/
sudo unzip -qq ${ZIP_NAME} -d /opt/

then
   sudo mkdir -p /usr/local/share/applications
   =>
   
   then
   echo sudo mkdir -p /usr/local/share/applications
   sudo mkdir -p /usr/local/share/applications
Users will feel informed and this is a good thing. Also script uses pattern RookLift used for naming, some changes of this pattern will break the script. But overall core is based on Rooklift GitHub, then they get version name by dropping not needed first part or last part - its safe, its based on Rooklift nibbler GitHub link.

File: src/install.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
```
Comment (vondele) [inline src/install.sh:2]: I would suggest to use `set -e` or install an error handler
```bash
error()
{
  echo "installation failed on line $1"
  exit 1
}
trap 'error ${LINENO}' ERR
```

**Comment by cyrillkuettel** [comment]
There were some good suggestions mentioned here. Thanks.
I've implemented every idea.

- [x] directory `/tmp` is used for extraction instead of ~/Downloads as suggested by @rooklift
- [x] Explaining why sudo is needed, suggested by @teck45
- [x] suggested by @vondele: add `set -e` to exit immediately if a command exits with a non-zero status 
- [x] check if curl installed (you never know)

One fundamental assumption: the naming convention is not subject to change: `nibbler-2.3.6-linux.zip`

**Comment by rooklift** [comment]
Alright, thanks everyone

**Pull request merged**